stake-pool/js/src/schema.ts

```diff
@@ -0,0 +1,181 @@
+import {Schema, serialize, deserializeUnchecked} from 'borsh';
+import BN from 'bn.js';
+
+// Class wrapping a plain object
+export abstract class Assignable {
```

Comment:
You can remove this and use `Struct` https://github.com/solana-labs/solana/blob/ec1a307a7c2618b9bdea7b82a39a36ba194ae0d0/web3.js/src/util/borsh-schema.ts#L5

Reply:
I need to use `deserializeUnchecked` in the `decode` method or validatorlist won't decode properly. Is it possible to change this in `borsh-schema.ts`?

Reply:
Ah right very good point.  Could you add another function `decodeUnchecked` in `borsh-schema.ts` which calls `deserializeUnchecked` under the hood?  Tag me on the PR and I'll review it as quickly as possible.